packages/flutter/lib/src/material/bottom_sheet.dart

```diff
@@ -431,20 +430,20 @@ class _BottomSheetState extends State<BottomSheet> {
 
 // See scaffold.dart
 
-typedef _SizeChangeCallback<Size> = void Function(Size size);
+typedef _SizeChangeCallback = void Function(Size size);
```

Comment:
The original typedef was quite interesting! 😅

I'm wondering… could we just remove this and use `ValueChanged<Size>`?